src/mon/ConfigMonitor.cc

```diff
@@ -574,19 +574,18 @@ void ConfigMonitor::load_config()
   *_dout << dendl;
 
   // refresh our own config
-  {
-    const OSDMap& osdmap = mon->osdmon()->osdmap;
-    map<string,string> crush_location;
-    osdmap.crush->get_full_location(g_conf->host, &crush_location);
-    map<string,string> out;
-    config_map.generate_entity_map(
-      g_conf->name,
-      crush_location,
-      osdmap.crush.get(),
-      string(), // no device class
-      &out);
-    g_conf->set_mon_vals(g_ceph_context, out);
-  }
+  const OSDMap& osdmap = mon->osdmon()->osdmap;
+  map<string,string> crush_location;
+  osdmap.crush->get_full_location(g_conf->host, &crush_location);
+  map<string,string> out;
+  config_map.generate_entity_map(
+    g_conf->name,
+    crush_location,
+    osdmap.crush.get(),
+    string(), // no device class
+    &out);
+  g_conf->set_mon_vals(g_ceph_context, out);
+  g_conf->apply_changes(NULL);
```

Comment:
This should have been a one line addition. Why are we changing the remaining 13 lines?

Reply:
because of format cleanup. I deleted the" { }"

Reply:
Yes. And I thank you for your effort cleaning up the code. I just think it's unnecessary given the small change you are making.

Reply:
i concur with Joao. could you drop the "format cleanup" from this PR? it would be easier to identify the change. we can always reindent the code when this part is rewritten.

Reply:
@jecluis @tchaikov format cleanup dropped, please take a look.

Reply:
@jecluis ping.